homeassistant/components/pooldose/strings.json

```diff
@@ -0,0 +1,149 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Set up SEKO PoolDose device",
+        "description": "The device password must be set to default (0000).",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        },
+        "data_description": {
+          "host": "IP address or hostname of your device"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "api_not_set": "API version not found in device response",
+      "api_not_supported": "Unsupported API version {api_version_is} (shall: {api_version_should})",
```

Comment:
shall? I would assume that that is the expected version, I think we can get some better wording